elf/module.go

```diff
@@ -457,6 +458,12 @@ func (b *Module) closeMaps() error {
 	return nil
 }
 
+// CloseOptions can be used for custom `Close` parameters
+type CloseOptions struct {
+	// Set Unpin to true to close pinned maps as well
+	Unpin bool
```

Comment:
ok, I guess it doesn't matter and it's done only for small comments (like https://golang.org/src/go/doc/example.go) :+1: